Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml

```diff
@@ -46,6 +46,23 @@ configuration:
   name: secret_key
   required: false
   type: 4
+- display: Timeout
+  name: timeout
+  additionalinfo: The time in seconds till a timeout exception is reached. You can
+    specify just the read timeout (for example 60) or also the connect timeout followed
+    after a comma (for example 60,10). If a connect timeout is not specified, a default
+    of 10 second will be used.
+  defaultvalue: 60,10
+  required: false
+  type: 0
+- display: Retries
+  name: retries
+  defaultvalue: 5
+  additionalinfo: The maximum retry attempts when connection or throttling errors
+    are encountered. Set to 0 to disable retries. The default value is 5 and the limit is for 10 retries.
+    Please considered when you increase the number of retries, execution time will increased respectively.
```

Comment:
```suggestion
  additionalinfo: The maximum retry attempts when connection or throttling errors
    are encountered. Set to 0 to disable retries. The default value is 5 and the limit is 10.
    When increasing the number of retries, execution time will increased respectively.
```

Reply:
@ShirleyDenkberg reword it, now the final description should apperar